### PR TITLE
front: fix open data import

### LINKED ISSUE #9315: Can't import trains from openData
### What happened?

We can't create trainSchedules from openData imported trains anymore. 

How to resolve : 

- create a new prop for Viriato imported files, so that don't check invalid ch when trains are imported from openData. 

### What did you expect to happen?

_No response_

### How can we reproduce it (as minimally and precisely as possible)?

1. Go to any scenario
2. Click import
3. Enter an origin and destination - Search for trains
4. Import the trains

### On which environments the bug occurs?

Recette (SNCF), Dev (SNCF), Local

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

.

### PR DESCRIPTION
**Pull request by RomainValls**
closes #9315 

How to test : 
- Enter origin / destination in the import tab for openData import
- Export the timetable into a json file, and try importing it.
- Try importing Viriato files with the files in the link : https://sncf.sharepoint.com/sites/OSRD644GrpO365/Documents%20partages/Forms/AllItems.aspx?id=%2Fsites%2FOSRD644GrpO365%2FDocuments%20partages%2FDonn%C3%A9es%2F3%2DFichiers%20Viriato&viewid=21cdca8b%2D1763%2D4a6e%2Da9e3%2Dc9c036f6a510

### CODE REVIEW

File: front/src/applications/operationalStudies/views/ImportTrainSchedule.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ const ImportTrainSchedule = ({
   const [trainsList, setTrainsList] = useState<ImportedTrainSchedule[]>([]);
   const [isLoading, setIsLoading] = useState(false);
   const [trainsJsonData, setTrainsJsonData] = useState<TrainScheduleBase[]>([]);
+  const [trainsViriatoData, setTrainsViriatoData] = useState<ImportedTrainSchedule[]>([]);
```
Comment (emersion) [inline front/src/applications/operationalStudies/views/ImportTrainSchedule.tsx]: I'd suggest renaming Virtiato to XML or something

**Review by Synar — COMMENTED** [review]
checkFirstStepUIC is also used to know if ch should be checked for each step, should it have a more explicit name? For example checkChAndFirstStepUIC?

**Comment by Synar** [comment]
Otherwise the code lgtm, though I haven't tested it

**Review by Synar — APPROVED** [review]
LGTM!

**Review by Math-R — APPROVED** [review]
lgtm !

**Pull request merged**